libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.h

```diff
@@ -22,16 +27,38 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#import "CSFSalesforceAction.h"
+#import <Foundation/Foundation.h>
+
+@interface SFNetwork : NSObject <NSURLSessionDataDelegate, NSURLSessionDownloadDelegate, NSURLSessionStreamDelegate>
+
+typedef void (^SFDataResponseBlock) (NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error);
+typedef void (^SFDownloadResponseBlock) (NSURL * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error);
+
+@property (nonatomic, readonly, strong, nonnull) NSURLSession *ephemeralSession;
+@property (nonatomic, readonly, strong, nonnull) NSURLSession *backgroundSession;
+@property (nonatomic, readwrite, assign) BOOL useBackground;
```

Comment:
This property allows the app to pick whether to use a background session or not.